docs/distributions/gentoo/installation.md

```diff
@@ -1,89 +1,42 @@
 # Installing Gentoo Linux on a T2 Mac
 
-!!! Warning "Unmaintained distro"
-    Gentoo is currently unmaintained. Due to lack of any maintainer, the Gentoo guides shall not be updated unless we get another maintainer. If you want to be a maintainer for Gentoo, contact the [Discord](https://discord.com/invite/68MRhQu).
-
-## Hardware Requirements
```

Comment:
Why were the hardware requirements removed?

Reply:
This is covered in the pre-install guide, is it not?
"For other distros, you can download the distro's normal install ISO, but you will have to use an external USB keyboard and mouse for this install process, and you may need a wired internet connection. Additionally, if you later have issues with installing the bootloader (Eg :- GRUB), try booting the ISO with the efi=noruntime kernel parameter."

Reply:
Oh, my bad.